lib/cheerio.js

```diff
@@ -40,6 +40,7 @@ var Cheerio = (module.exports = function (selector, context, root, options) {
     return new Cheerio(selector, context, root, options);
   }
 
+  this.length = 0;
```

Comment:
Remove the prototype property from L109?

Reply:
Done